workflow/snakemake_rules/main_workflow.smk

```diff
@@ -1128,6 +1128,38 @@ rule logistic_growth:
             --output {output.node_data} 2>&1 | tee {log}
         """
 
+rule mutational_fitness:
+    input:
+        tree = "results/{build_name}/tree.nwk",
+        alignments = lambda w: rules.aa_muts_explicit.output.translations,
+        distance_map = config["files"]["mutational_fitness_distance_map"]
+    output:
+        node_data = "results/{build_name}/mutational_fitness.json"
+    benchmark:
+        "benchmarks/mutational_fitness_{build_name}.txt"
+    conda:
+        config["conda_environment"]
+    log:
+        "logs/mutational_fitness_{build_name}.txt"
+    params:
+        genes = ' '.join(config.get('genes', ['S'])),
+        compare_to = "root",
+        attribute_name = "mutational_fitness"
```

Comment:
These parameters are all candidates for a new top-level `mutational_fitness` section of the configuration. We can leave them hardcoded here, but it could be worth the extra effort to expose them as config parameters as we think about exporting our workflow to Terra and other platforms.

Reply:
I'm actually less sure about this after thinking about this more. This suggestion is probably better for a future PR.

Reply:
Good thing to flag. I can imagine a future PR with things like how many weeks to project tip frequencies ahead etc... where there are more configuration options to expose. Or even whether to use Obermeyer et al mutational mapping JSON or some other JSON. But I'd think okay to not be in scope for this PR.